WORKSPACE

```diff
@@ -62,9 +62,24 @@ http_archive(
 
 http_archive(
     name = "googleapis_git",
-    sha256 = "0eaf8c4d0ea4aa3ebf94bc8f5ec57403c633920ada57a498fea4a8eb8c17b948",
-    strip_prefix = "googleapis-68c72c1d1ffff49b7d0019a21e65705b5d9c23c2",  # 06/04/2020
-    url = "https://github.com/googleapis/googleapis/archive/68c72c1d1ffff49b7d0019a21e65705b5d9c23c2.tar.gz",
+    sha256 = "0e33a2f12ffb8da6a296d9c30582f01d4ad9e3ecd35aae11b50d4ed3a855946e",
+    strip_prefix = "googleapis-923a4b9a6538087654f02c089cbbaa7c45fd06bd",  # 10/19/2022
+    url = "https://github.com/googleapis/googleapis/archive/923a4b9a6538087654f02c089cbbaa7c45fd06bd.tar.gz",
+)
+
+http_archive(
+    name = "com_google_absl",
+    urls = [
+        "https://github.com/abseil/abseil-cpp/archive/refs/tags/20210324.2.tar.gz",
+    ],
+    patches = [
+        "@//third_party:com_google_absl_f863b622fe13612433fdf43f76547d5edda0c93001.diff"
+    ],
+    patch_args = [
+        "-p1",
+    ],
```

Comment:
I don't understand why we would need to patch a release? Also not too sure where is `@//third_party` referring to?
Also it seems `patches` would conflict with `patch_args` as explained in https://bazel.build/rules/lib/repo/http#http_archive-patches
> patches: list of files that are to be applied as patches after extracting the archive. By default, it uses the Bazel-native patch implementation which doesn't support fuzz match and binary patch, but Bazel will fall back to use patch command line tool if `patch_tool` attribute is specified or there are arguments other than `-p` in `patch_args` attribute.

Reply:
Haven't fully looked into it, was playing around with this import, and it should be removed as mentioned.